lib/exlasticsearch/model.ex

```diff
@@ -80,14 +81,19 @@ defmodule ExlasticSearch.Model do
       def __es_index__(:read), do: index_version(unquote(type), @read_version)
       def __es_index__(:index), do: index_version(unquote(type), @index_version)
       def __es_index__(:delete), do: __es_index__(:read)
+      def __es_index__(:read_sync_test), do: __es_index__(:read) <> "_sync"
```

Comment:
This is the only part that is weird to me, and will not be necessary after we confirm the new system works and we switch over. Maybe there is a better way I can't think of right now.

Reply:
made an edit so you can specify a custom index by passing in the non standard `:read`, `:index` and `:delete`.